### ISSUE TITLE
Allow hyperlinks

### ISSUE THREAD

**Issue by UltimateGeek** [ArnoldSmith86/virtualtabletop#79]
```gherkin
Feature: Hyperlinks
  As a game organizer
  I need a clickable hyperlink in the game room
  So that I have a visual reminder to go do something outside of the game 
  And so that I can quickly go to that other something (e.g. to capture scores in a tracker when the game is over)

Scenario: click hyperlink with confirmation
  Given a hyperlink configured to use a confirmation
  When I click the hyperlink
  Then it prompts me to confirm that I wish to go to the other site
  And it shows me the full URL

Scenario: confirm hyperlink activation
  Given I click the hyperlink
  And it prompts me to confirm that I wish to go to the other site
  When I confirm to activate the hyperlink
  Then it launches the URL in a new browser tab

Scenario: click hyperlink without confirmation
  Given a hyperlink configured to not use a confirmation
  When I click the hyperlink
  Then it launches the URL in a new browser tab
```

**Comment by larkob** [comment]
I was proposeing markdown support for labels (see https://github.com/ArnoldSmith86/virtualtabletop/issues/36), so a link could be written as `[link name](link_url)` in a Label widget.

As for confirmation, I'm not sure what's the best way, but I wonder if this is really so important?

**Comment by UltimateGeek** [comment]
A confirmation is important for security reasons, especially with there being no authentication.

**Comment by robartsd** [comment]
I don't think a confirmation is important if the url is in the href attribute of the link (I would want it if the href attribute is not the url). We'd probably would want to use` rel="noopener noreferrer nofollow"` on any link created from user submitted content.

**Comment by RaphaelAlvez** [comment]
It was already agreed on meeting that if url come to be supported there should be some sort of confirmation.
As always, decisions made in meetings are not set in stone but if @robartsd or anyone wants to something else I think you should provide some arguments to promote their point of view.

The main arguments shown in the meeting were that this would be a security feature and that it would avoid spam.

**Comment by bjalder26** [comment]
#582 is related

**Comment by 96LawDawg** [comment]
Implemented (for basic widgets) in #1889
